src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -225,14 +271,12 @@ const styles = StyleSheet.create({
   }),
   navContainer: (isHeaderHide, top) => ({
     flexDirection: 'row',
-    height:
-      (isHeaderHide
-        ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2
-        : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT) + top,
+    height: isHeaderHide
+      ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2
+      : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT,
     justifyContent: 'space-between',
     alignItems: 'flex-start',
     backgroundColor: 'white',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You've removed the `paddingTop` style property from the `navContainer` style. Ensure that this does not affect the layout of your navigation container adversely.


<!-- This is an auto-generated comment by OSS CodeRabbit -->